docs/fides/docs/installation/installation.md

```diff
@@ -0,0 +1,66 @@
+# Installation
+
+This page describes installation options that you might use when considering how to install fidesctl. Fidesctl consists of multiple components, possibly distributed among various physical or virtual machines. Therefore installation of fidesctl might be complex, depending on the options you choose.
+
+You should also check-out the [Prerequisites](prerequisites_dependencies.md) that must be fulfilled when installing fidesctl. Fidesctl requires additional [Dependencies](prerequisites_dependencies.md) to be installed - which can be done via `extras`.
+
+When you install fidesctl, you need to [setup the database](database.md) which must also be kept updated when fidesctl is upgraded.
+
+## Using PyPI
+
+More details: [Installation from PyPI](pypi.md)
+
+**When this option works best**
+
+* This installation method is useful when you are not familiar with Containers and Docker. and want to install fidesctl on physical or virtual machines and you are used to installing and running software using custom deployment mechanism.
```

Comment:
casing and stray period.

```suggestion
* This installation method is useful when you are not familiar with containers or Docker and want to install fidesctl on physical or virtual machines and you are used to installing and running software using custom deployment mechanism.
```

With those changes, it's still a long sentence. Should we break it up into sub bullets?